apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/pathAssertions.kt

```diff
@@ -254,6 +254,27 @@ fun <E> path(path: String, assertionCreator: Expect<E>.() -> Unit): PathWithCrea
 infix fun <T : Path> Expect<T>.toBe(@Suppress("UNUSED_PARAMETER") readable: readable): Expect<T> =
     _logicAppend { isReadable() }
 
+//TODO move to pathExpectations.kt with 0.18.0
+/**
+ * Expects that the subject of `this` expectation (a [Path]) is not readable;
+ * meaning that there is a file system entry at the location the [Path] points to and
+ * that the current thread does not have the permission to read from it.
+ *
+ * This assertion _resolves_ symbolic links.
+ * Therefore, if a symbolic link exists at the location the subject points to,
+ * search will continue at the location the link points at.
+ *
+ * This assertion is not atomic with respect to concurrent file system operations on the paths the assertion works on.
+ * Its result, in particular its extended explanations, may be wrong if such concurrent file system operations
+ * take place.
+ *
+ * @return an [Expect] for the subject of `this` expectation.
+ *
```

Comment:
Can you please also create a corresponding sample in ch.tutteli.atrium.api.infix.en_GB.samples.PathExpectationSamples (file does not exist, see api-fluent for an example) and add the `@sample` link here:
```suggestion
 * @sample ch.tutteli.atrium.api.infix.en_GB.samples.PathExpectationSamples.notToBeReadable
 *
```